pkg/permission/permission.go

```diff
@@ -1,10 +1,30 @@
 package permission
 
 import (
+	"context"
+	"encoding/json"
+
 	"github.com/OpenSlides/openslides-permission-service/internal/core"
-	"github.com/OpenSlides/openslides-permission-service/pkg/definitions"
 )
 
-func New(dataprovider definitions.ExternalDataProvider) definitions.Permission {
+// New returns a new permission service.
+func New(dataprovider ExternalDataProvider) Permission {
 	return core.NewPermissionService(dataprovider)
 }
+
+// ExternalDataProvider is the connection to the datastore. It returns the data
+// required by the permission service.
+type ExternalDataProvider interface {
+	// If a field does not exist, it is not returned.
+	Get(ctx context.Context, keys ...string) (map[string]json.RawMessage, error)
```

Comment:
Please do not add `ctx` here, since if you want to propagate it from the permission interface, it is all over the place. You can manage the context outside of this ocde wiht the provided dataprovider.